webserver/errors.py

```diff
@@ -1,13 +1,12 @@
 from flask import render_template, jsonify
-import webserver.exceptions
+from webserver.views.api import exceptions as api_exceptions
```

Comment:
I'm not a fan of renaming modules when importing them. Can we just refer to exceptions as `webserver.views.api.exceptions.APIError` ?